static/locales/en-US.yaml

```diff
@@ -576,6 +576,8 @@ Settings:
   Proxy Settings:
     Proxy Settings: Proxy Settings
     Enable Tor / Proxy: Enable Tor / Proxy
+    Proxy Login: Proxy Login
```

Comment:
Not blocking but I see most articles online refer it as username instead of login